web/tests/EducationBenchmarking.Web.E2ETests/Pages/BenchmarkWorkforcePage.cs

```diff
@@ -57,7 +59,7 @@ public async Task AssertPage()
             await chart.ShouldBeVisible();
         }
 
-        var schoolWorkForceExpectedOptions = new[] { "total", "headcount per FTE", "pupils per staff role" };
+        var schoolWorkForceExpectedOptions = new[] { "total", "pupils per staff role" };
```

Comment:
the dropdown should have 'headcount per FTE' in it as per the requirement.

Reply:
Recheck the user story - the "Headcount per FTE" option is not present.

For school workforce chart the value when this dimension is select will always be 1 (School workforce / school workforce).

Reply:
just double checked the story and current implemented system.  for school workforce FTE -> 'the percentage of workforce' option should not present in the dropdown as it would always be 1.
and for school workforce(headcount) -> 'head count per FTE' should not be present in the dropdown.